SmartEVSE-3/include/evse.h

```diff
@@ -534,6 +540,7 @@ const struct {
     {"SUM MAINS","Capacity Rate limit on sum of MAINS Current (A)",    10, 600, MAX_SUMMAINS},
     {"", "Hold 2 sec to stop charging", 0, 0, 0},
     {"", "Hold 2 sec to start charging", 0, 0, 0},
+    {"OCPP",    "Select OCPP mode",                                   0, 1, OCPP_MODE},
```

Comment:
Please move this line 2 positions up, just after "SUM MAINS", to keep all standard menu items together.